.github/workflows/node.js.yml

```diff
@@ -7,8 +7,8 @@ on:
       - master
       - develop
       - 'run-ci/**'
-  pull_request_target:
-    branches: [ master, develop, test-pr ]
+  pull_request:
+    branches: [ master, develop, test-pr, feat/simplifyci ]
```

Comment:
you want to keep the `feat/simplifyci` branch mentioned here for a reason?

Reply:
No, good catch :) That was for testing